<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabVisualizzaDatiPersonali</class>
 <widget class="QTabWidget" name="tabVisualizzaDatiPersonali">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>457</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Struttura Alberghiera</string>
  </property>
  <widget class="QWidget" name="VisualizzaDatiPersonaliReceptionist">
   <attribute name="title">
    <string>Visualizza dati personali</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupboxSchedaVisualizzazioneDatiPersonaliReceptionist">
      <property name="title">
       <string>Scheda dati personali:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineeditIndirizzoEmail">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>myEmail@domain.com</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDateEdit" name="dateeditDataDiNascita">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineeditCognome">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Cognome</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineeditLuogoDiNascitaProvincia">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Luogo di nascita(Provincia)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLuogoDiNascitaProvincia">
         <property name="text">
          <string>Luogo di nascita(provincia):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCognome">
         <property name="text">
          <string>Cognome:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelTurnoDiLavoro">
         <property name="text">
          <string>Turno di lavoro:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelTelefono">
         <property name="text">
          <string>Telefono:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineeditIBAN">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>IBAN</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="lineeditTurnoDiLavoro">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Mattina/Pomeriggio</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineeditNome">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelIndirizzoEmail">
         <property name="text">
          <string>Indirizzo Email:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelDataDiNascita">
         <property name="text">
          <string>Data di nascita:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineeditTelefono">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Telefono</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelIBAN">
         <property name="text">
          <string>IBAN:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelNome">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelRuolo">
         <property name="text">
          <string>Ruolo:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="lineeditRuolo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Ruolo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
